infra/main.parameters.json

```diff
@@ -9,10 +9,13 @@
       "value": "${AZURE_LOCATION}"
     },
     "useMonitoring": {
-      "value": "${USE_MONITORING}"
+      "value": "${USE_MONITORING=yes}"
```

Comment:
It would be great if we remove this default value setting part, so that devs will be prompted to enter the name, unless they manually run `azd env set`.